api/connectors/read

```diff
@@ -27,6 +27,7 @@ use JSON;
 
 require '/usr/libexec/nethserver/api/lib/helper_functions.pl';
 require '/usr/libexec/nethserver/api/nethserver-mail/lib/mail_functions.pl';
+use IPC::Run 'run';
```

Comment:
We also have a similar function that does not spawn a shell, but does not capture stderr:

https://github.com/NethServer/nethserver-cockpit/blob/ad17568fa17167752323b87d4ac1254a68f3e7c2/lib/perl/NethServer/ApiTools.pm#L138

However a stderr-capturing variant is welcome!

----

side note: you could replace the `require helper_functions.pl` line above with `use NethServer::ApiTools;`

Reply:
In this case stderr it's not really needed, I'm going to use the cool `exec_slurp` function!